doc/vim-rspec.txt

```diff
@@ -0,0 +1,106 @@
+*vim-rspec.txt* run RSpec from within vim
+
+INTRODUCTION                                    *vim-rspec*
+
+A lightweight RSpec runner for Vim and MacVim.
+
+Repo:    https://github.com/thoughtbot/vim-rspec
+
+FUNCTIONS                                       *vim-rspec-functions*
+
+vim-rspec provides four functions for executing your specs:
+
+                                                *all-specs* *RunAllSpecs()*
+:RunAllSpecs()          Run all spec files in the project.
+                        Equivalent to running
+
+                            rspec spec
```

Comment:
Yeah  - could be. My thinking was that this was slightly more explicit, but I'm happy to change it.